armada_backend/supervisor/docker_client.conf

```diff
@@ -0,0 +1,4 @@
+[program:install_docker]
+directory=/opt/armada-docker/armada_backend
+command=python install_docker_client.py
+autorestart=false
```

Comment:
I think we should use autorestart, for example in case downloading fails.
But instead I would use `startsecs=0`, so when the download is really fast, it won't be retried.